assets/js/components/PremiumPill.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function PremiumPill() {
```

Comment:
We are missing `className` as a prop here. Keep in mind that having `className` as a prop always allows for better composability into the DOM

(for example having more of these inline could allow us to specify individual margins or similar stuff)

Reply:
Nice catch, binding classNames with a prop is always a good Idea +1

Reply:
Done